examples/sql_bank/main.go

```diff
@@ -26,14 +26,18 @@ import (
 
 	"github.com/lib/pq"
 
+	"github.com/cockroachdb/cockroach/security"
+	"github.com/cockroachdb/cockroach/security/securitytest"
+	"github.com/cockroachdb/cockroach/server"
 	"github.com/cockroachdb/cockroach/util/log"
 )
 
 const (
 	maxTransfer = 999
 	numAccounts = 999
-	concurrency = 25
+	concurrency = 5
 	aggregate   = true
+	usePostgres = false
```

Comment:
You should make this a flag. The `flags` package is really easy to use. Heck, these should all be flags, but at least the `usePostgres` one to start.